Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -179,15 +177,14 @@ public void run() {
             adHeightInDp = (int) adHeightObject;
         }
 
-        if((adWidthInDp == VUNGLE_MREC.getWidth() && adHeightInDp == VUNGLE_MREC.getHeight())
-                || (adWidthInDp == 336 && adHeightInDp == 280)) {
+        if ((adWidthInDp >= VUNGLE_MREC.getWidth() && adHeightInDp >= VUNGLE_MREC.getHeight())) {
             adSizeType = VUNGLE_MREC;
-        } else if (adWidthInDp == BANNER_SHORT.getWidth() && adHeightInDp == BANNER_SHORT.getHeight()) {
-            adSizeType = BANNER_SHORT;
-        } else if (adWidthInDp == BANNER.getWidth() && adHeightInDp == BANNER.getHeight()) {
-            adSizeType = BANNER;
-        } else if (adWidthInDp == BANNER_LEADERBOARD.getWidth() && adHeightInDp == BANNER_LEADERBOARD.getHeight()) {
+        } else if (adWidthInDp >= BANNER_LEADERBOARD.getWidth() && adHeightInDp >= BANNER_LEADERBOARD.getHeight()) {
             adSizeType = BANNER_LEADERBOARD;
+        } else if (adWidthInDp >= BANNER.getWidth() && adHeightInDp >= BANNER.getHeight()) {
+            adSizeType = BANNER;
+        } else {
+            adSizeType = BANNER_SHORT;
```

Comment:
@HuFangshuai  @palrahul isn't it will be always BANNER_SHORT and NETWORK_NO_FILL can never happen for small sizes?
see https://github.com/mopub/mopub-android-mediation/blob/master/AdMob/src/main/java/com/mopub/mobileads/GooglePlayServicesBanner.java#L189
https://github.com/Vungle/mopub-android-mediation/pull/16/files#diff-87bfb1fb1eb3bd3c6c726f1d94fc22f1L116

Reply:
@alexander-gubov Yes, I noticed about it for admob, and the purpose is trying to cover all AdSize. see https://github.com/mopub/mopub-android-mediation/blob/7009ea20d0010a66507fd2886d000c887b834a28/FacebookAudienceNetwork/src/main/java/com/mopub/mobileads/FacebookBanner.java#L229